### PR TITLE
Remove serialization of custom objects

### PR DESCRIPTION
**Pull request by jiivan**
fixes: #101

### CODE REVIEW

**Comment by jiivan** [comment]
@Krigpl will be resolved after creating PR `Upgrade golem-messages to 1.7` in golem core.

See for example https://github.com/golemfactory/golem/pull/1974/

**Comment by Krigpl** [comment]
> @Krigpl will be resolved after creating PR Upgrade golem-messages to 1.7 in golem core.

This is a weird order, usually you'd first remove usages and only when the offending code is no longer used anywhere you'd delete it.
What if it turns out that refactoring this dependency is more annoying than expected and will be delayed in time? And in the meantime some other feature gets added to this module and we'd like to update the version in golem core? We wouldn't be able to do that.

**Comment by jiivan** [comment]
@Krigpl looks like there is no issue here, as nodelist sent in `Peers` is a list of dicts: https://github.com/golemfactory/golem/blob/develop/golem/network/p2p/p2pservice.py#L592

**Comment by Krigpl** [comment]
Except that this value is a Python struct https://github.com/golemfactory/golem/blob/develop/golem/network/p2p/p2pservice.py#L588

**Comment by jiivan** [comment]
@Krigpl good catch. Fixed in https://github.com/golemfactory/golem/pull/2013

File: golem_messages/serializer.py [hunk 1]
```diff
@@ -17,7 +12,7 @@
 logger = logging.getLogger('golem.core.simpleserializer')
 
 
-OBJECT_TAG = 239
+OBJECT_TAG = 239  # unused, old protocol
```
Comment (Krigpl) [inline golem_messages/serializer.py]: Reason for keeping it?

**Pull request merged**